toolkit/scripts/toolchain/container/toolchain-remote-wget-list

```diff
@@ -1,59 +1,59 @@
 https://ftp.gnu.org/gnu/autoconf/autoconf-2.71.tar.xz
-http://ftp.gnu.org/gnu/automake/automake-1.16.5.tar.gz
+https://ftp.gnu.org/gnu/automake/automake-1.16.5.tar.gz
 https://ftp.gnu.org/gnu/binutils/binutils-2.37.tar.xz
 https://sourceware.org/ftp/debugedit/5.0/debugedit-5.0.tar.xz
-http://ftp.gnu.org/gnu/dejagnu/dejagnu-1.6.2.tar.gz
+https://ftp.gnu.org/gnu/dejagnu/dejagnu-1.6.2.tar.gz
 https://sourceware.org/ftp/elfutils/0.186/elfutils-0.186.tar.bz2
 https://ftp.gnu.org/gnu/findutils/findutils-4.8.0.tar.xz
 https://github.com/westes/flex/releases/download/v2.6.4/flex-2.6.4.tar.gz
 https://ftp.gnu.org/gnu/gdbm/gdbm-1.21.tar.gz
-http://ftp.gnu.org/gnu/gmp/gmp-6.2.1.tar.xz
-http://ftp.gnu.org/gnu/gperf/gperf-3.1.tar.gz
+https://ftp.gnu.org/gnu/gmp/gmp-6.2.1.tar.xz
+https://ftp.gnu.org/gnu/gperf/gperf-3.1.tar.gz
 https://github.com/libffi/libffi/releases/download/v3.4.2/libffi-3.4.2.tar.gz
-http://ftp.gnu.org/gnu/libtool/libtool-2.4.6.tar.xz
+https://ftp.gnu.org/gnu/libtool/libtool-2.4.6.tar.xz
 https://ftp.gnu.org/gnu/m4/m4-1.4.19.tar.gz
 https://ftp.gnu.org/gnu/make/make-4.3.tar.gz
 https://www.kernel.org/pub/linux/docs/man-pages/Archive/man-pages-5.02.tar.xz
 https://ftp.gnu.org/gnu/mpc/mpc-1.2.1.tar.gz
-http://www.mpfr.org/mpfr-4.1.0/mpfr-4.1.0.tar.xz
+https://www.mpfr.org/mpfr-4.1.0/mpfr-4.1.0.tar.xz
 https://invisible-mirror.net/archives/ncurses/ncurses-6.2.tar.gz
-http://ftp.gnu.org/gnu/patch/patch-2.7.6.tar.xz
+https://ftp.gnu.org/gnu/patch/patch-2.7.6.tar.xz
 https://pkg-config.freedesktop.org/releases/pkg-config-0.29.2.tar.gz
 https://sourceforge.net/projects/procps-ng/files/Production/procps-ng-3.3.17.tar.xz
 https://www.python.org/ftp/python/3.9.13/Python-3.9.13.tar.xz
 https://tukaani.org/xz/xz-5.2.5.tar.xz
-https://www.zlib.net/zlib-1.2.12.tar.xz
+https://src.fedoraproject.org/repo/pkgs/zlib/zlib-1.2.12.tar.xz/sha512/12940e81e988f7661da52fa20bdc333314ae86a621fdb748804a20840b065a1d6d984430f2d41f3a057de0effc6ff9bcf42f9ee9510b88219085f59cbbd082bd/zlib-1.2.12.tar.xz
```

Comment:
I'm a bit curious why the zlib location has changed so much but the spec still refers to zlib.net

Reply:
The original location "https://www.zlib.net/zlib-1.2.12.tar.xz" stopped providing version 1.2.12 after 1.2.13 was released. I found an alternate server at fedoraproject.org that supports https, so made the switch.